packages/foundry/contracts/FYToken.sol

```diff
@@ -86,6 +86,28 @@ contract FYToken is IFYToken, IERC3156FlashLender, AccessControl, ERC20Permit, C
         emit FlashFeeFactorSet(flashFeeFactor_);
     }
 
+    ///@dev Converts the amount of the principal to the underlying
```

Comment:
note for above: @alcueca mentioned we want to start moving the interfaces out of that `vault-interfaces` repo and into the repo of the contract.  this might be a time to start that and move the IFYToken file to this repo

Reply:
Created.